src/components/CarCard.tsx

```diff
@@ -0,0 +1,223 @@
+import Image from './Image';
+import Condition from './Condition';
+import styled from 'styled-components';
```

Comment:
you should configure your linter to keep external libraries import at the top of the file, and also we should have a blank line after all the import statements